cairo/tests/legacy/utils/test_bytes_legacy.py

```diff
@@ -146,3 +146,8 @@ class TestBytesToFelt:
         def test_should_convert_bytes_to_felt_with_overflow(self, cairo_run, data):
             output = cairo_run("test__bytes_to_felt", data=list(data))
             assert output == int.from_bytes(data, byteorder="big") % DEFAULT_PRIME
+
+        @given(data=binary(min_size=0, max_size=35))
```

Comment:
The assumption is max 31 bytes. 
Should this raise?

Reply:
No, this just wraps around. I'm mirroring the already existing function and test right above